### DISCUSSION TITLE
Changing Font on Time Display

### DISCUSSION THREAD

**Discussion by DanielHirunrusme** [muxinc/elements#654] [Q&A] [answered]
I'm trying to change the time display font from Arial. When I inspect the time display I can see it's being set in the :host pseudo element. I can change the font in the title. However, I'm having trouble overriding the time display either with mux parts or host-context. Any tips n tricks on changing this font? 

![image](https://user-images.githubusercontent.com/1197940/230797609-f16e73b7-a837-4941-bcee-0e69e78f47d7.png)

`
:host {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    vertical-align: middle;
    box-sizing: border-box;
    background: var(--media-control-background, rgba(20,20,30, 0.7));
    padding: var(--media-control-padding, 10px);
    font-size: 14px;
    line-height: var(--media-text-content-height, var(--media-control-height, 24px));
    font-family: Arial, sans-serif;
    text-align: center;
    color: #ffffff;
    pointer-events: auto;
}
`

**Comment by luwes** [comment]
thanks for the feedback! this will probably be added in the form of a CSS var soon.
see https://github.com/muxinc/media-chrome/issues/453

**Comment by luwes** [comment, marked as answer]
This was added in Media Chrome and Mux Player as CSS vars:

```css
--media-font
--media-font-weight
--media-font-size
--media-font-family
```

https://github.com/muxinc/media-chrome/pull/528
